### PR TITLE
Updated to read the api key and api user from the taskrc 

### PR DESCRIPTION
**Pull request by ksonney**
This changes the scripts to use taskw and read the habitica keys from the .taskrc. It no longer requires the user to modify the scripts themselves, but it does require Taskwarrior API v2 now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,17 @@ Add todo task on Habitica.com when task are added and complete task on Habitica.
...
 
-## Install
+Or using your system package manager. For example, on Ubuntu (and Ubuntu-ish) Linux:
+      sudo apt install python-taskw
+
+## Install these hooks
 Download and unzip files
 
-	wget https://github.com/fplourde/Taskwarrior-habitica-hooks/archive/master.zip
+	wget https://github.com/ksonney/Taskwarrior-habitica-hooks/archive/master.zip
```
Comment (fplourde) [inline README.md]: Do not link to a different project. Make the modification and I will merge the code.

**Comment by fplourde** [comment]
Thank you for your support

**Review by fplourde — CHANGES_REQUESTED** [review]
Update the link and I will merge the request.

Thank you for your support

**Review by fplourde — APPROVED** [review]

**Comment by ksonney** [comment]
Awesome! You updated it before I saw the emails, so all good!

**Pull request merged**